pearcej/cppds
# Fix links for all images in chapter 1 in cppds-v2

# Description

I fixed all image linking issues in chapter one. Also, I added the asset file outside the pretext directory and deleted the extra assets folder that was already outside the pretext folder.
## Related Issue



fix #482
## How Has This Been Tested?
1. Make changes locally.
2. Verify the changes locally.
## Sample images
![image](https://github.com/pearcej/cppds/assets/117699634/f4e203ed-6ae0-4a39-88e4-c555cc0d42e9)
![image](https://github.com/pearcej/cppds/assets/117699634/4127f40c-dbdc-4454-ba90-3812708b988a)
![image](https://github.com/pearcej/cppds/assets/117699634/0dbec539-b324-4356-9b41-b2c2a5c8359d)
